package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "alist",
+  "version": "1.0.0",
+  "description": "Top list generator",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CodeURJC-DAW-2022-23/webapp1.git"
+  },
+  "keywords": [
+    "Toplist",
+    "Ranking"
+  ],
+  "contributors": [
```

Comment:
contributors did u change "authors" to "contributors"? wut

Reply:
Changed it according to [docs](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)